department-of-veterans-affairs/vets-website
# Pension 76768 Spouse info - update deprecated fields

## Summary

convert spouse info form page to use v3 web component patterns

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/76768

## Testing done

load form, proceed to Spouse Information
- submit with empty fields - should receive errors for each required field
- enter all required information - should submit without error
- answer 'Yes' to "[Spouse] is a Veteran?" - should reveal additional field for VA number

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
